test/copy.tilelive.test.js

```diff
@@ -395,3 +395,27 @@ test('copy omnivore to Frankfurt', function(t) {
     });
   });
 });
+
+test('copy omnivore to s3 encrypted with AWS KMS', function(t) {
+  var bucket = 'mapbox-eu-central-1';
+  var kmsKeyId = 'put your arn:aws:kms: key id here';
```

Comment:
You'd need the KMS key alias here, right?

Reply:
Yep - this is just the straight up alias right? `alias/mapbox-tile-copy-test-kms`

Reply:
Yep!